crates/hyperqueue/src/dashboard/ui/screens/home/screen.rs

```diff
@@ -77,3 +90,15 @@ pub fn vertical_chunks(constraints: Vec<Constraint>, size: Rect) -> Vec<Rect> {
         .direction(Direction::Vertical)
         .split(size)
 }
+
+pub fn horizontal_chunks_with_margin(
```

Comment:
It seems like it would be shorter to just inline `vertical_chunks` and `horizontal_chunks_with_margin`.